### ISSUE TITLE
OverflowMenu doesn't allow distinction between icon's aria-label and alt text

### ISSUE THREAD

**Issue by rstrangfeld** [IBM/carbon-components-react#2110]
In the OverflowMenu, the way to provide an aria-label text for the visible icon (as required for accessibility) is to specify it as the "iconDescription" for the OverflowMenu. However, when you do this, the text you supply is also used as the "alt" string for the icon. This means that it is also displayed as a tooltip when a user hovers the mouse over the icon.

Normally there is a distinction between aria-label text and alt text, so that you can meet accessibility requirements without adding a visible tooltip for all users. That distinction is missing in OverflowMenu. 

A good way to provide it would be to have two separate properties in OverflowMenu, instead of using "iconDescription" for both purposes.

**Comment by rstrangfeld** [comment]
It looks like stale bot is threatening to close this issue because of inactivity. This is a vote not to do that as we're still interested in a fix. What needs to happen to accomplish that?
Thanks!

**Comment by dakahn** [comment]
**A hoverable tooltip is a poor way of communicating information to your users and is not accessible**, that said because the icon for the overflow menu has no accompanying label and the way overflow menus are used on the modern web -- tucking away your menu in a drop down for aesthetics rather than actually having a menu bar that is too full to include anymore menu items -- we include it here.

An argument could be made  to make it optional (I don't think that argument is strong), but I don't know why the aria-label _would be different_ than the tooltip if it was included.

**Comment by rstrangfeld** [comment]
I agree completely that the hoverable tooltip is not a good way to communicate information to the users. Our designers really don't want one. We are required to have an aria-label for accessibility, and we do want that, but not the tooltip.

I think it's unusual for the existence of aria-labels to force the existence of tooltips, or for components to require that aria-label text be identical to tooltip text, but both things are happening here. Currently the only way to get an aria-label is to specify iconDescription, and as soon as we do that we get an unwanted tooltip with the same text.

Thanks!
